ui/app/src/state/actions/dialogs.ts

```diff
@@ -303,3 +304,23 @@ export const rtePickerActionResult = /*#__PURE__*/ createAction<{ path: string;
   'RTE_PICKER_ACTION_RESULT'
 );
 // endregion
+
+// region BrokenReferences Cancellation
+
+export const showBrokenReferencesDialog = /*#__PURE__*/ createAction<Partial<BrokenReferencesDialogStateProps>>(
+  'SHOW_BROKEN_REFERENCES_DIALOG'
+);
+
+export const closeBrokenReferencesDialog = /*#__PURE__*/ createAction<StandardAction>('CLOSE_BROKEN_REFERENCES_DIALOG');
+
+export const brokenReferencesDialogClosed = /*#__PURE__*/ createAction('BROKEN_REFERENCES_DIALOG_CLOSED');
+
+export const fetchBrokenReferences = /*#__PURE__*/ createAction('FETCH_BROKEN_REFERENCES');
+
+export const fetchBrokenReferencesFailed = /*#__PURE__*/ createAction('FETCH_BROKEN_REFERENCES_FAILED');
+
+export const updateBrokenReferencesDialog = /*#__PURE__*/ createAction<Partial<BrokenReferencesDialogStateProps>>(
+  'UPDATE_BROKEN_REFERENCES_DIALOG'
+);
```

Comment:
Add/ensure correct payload types for all. For example the "failed" typically receives the error, add type and make sure its the right one by actually testing it and seeing it through.